release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,79 @@
+
+### Important upgrade notes
+
+- Due to Azure limitations, upgraded clusters do not support multiple Availability Zones. In order for a cluster to be able to scale across multiple Availability Zones, it must be created using the v11.1.0 release.
+
+---
+
+### azure-operator v2.9.0
+
+#### Minor changes
+
+- Cluster scaling is less disruptive.
+
+#### Improved worker nodes' upgrade process
+
+With this release the worker nodes' upgrade process got some notable changes and it is now faster and more graceful. It works by mirroring the existing nodes with new ones and then draining the old nodes. Once all old nodes are drained, they are terminated. This way kubernetes controller manager and scheduler can move the running workload to new nodes while respecting configured scheduling directives for undisrupted service.
+
+In order to ensure continuous service during upgrade, the workload must be configured with multiple replicas and appropriate [pod disruption budget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).
+
+### Kubernetes v1.16.3
+
+- No change from the previous version
+
+### Calico v3.10.1
+
+- No change from the previous version
+
+### cert-exporter [v1.2.1](https://github.com/giantswarm/cert-exporter/blob/master/CHANGELOG.md#121-2019-12-24)
+
+- No change from the previous version
+
+### chart-operator [v0.11.4](https://github.com/giantswarm/chart-operator/releases/tag/v0.11.4)
+
+- Now handling timeouts by pulling chart tarballs.
+
+### cluster-operator [v0.22.1](https://github.com/giantswarm/cluster-operator/releases/tag/v0.22.1)
+
+- Upgrade to Nginx Ingress Controller App 1.3.0
+
+### coreDNS v1.6.5 ([Giant Swarm app v1.1.3](https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v113))
+
+- Updated HPA manifests for Kubernetes 1.16.
+
+### CoreOS Container Linux v2247.6.0
+
+- No change from the previous version
+
+### etcd v3.3.17
+
+- No change from the previous version
+
+### external-dns v0.5.11 ([Giant Swarm app v1.1.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#v110))
+
+- No change from the previous version
+
+### Helm v2.16.1 (primarily for Giant Swarm internal use)
+
+- No change from the previous version
+
+### kube-state-metrics v1.9.2 ([Giant Swarm app v1.0.2](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#v102))
+
+- No change from the previous version
+
+### net-exporter [v1.6.0](https://github.com/giantswarm/net-exporter/blob/master/CHANGELOG.md#160-2020-01-29)
+
+- Allow disabling DNS TCP check.
+
+### nginx-ingress-controller v0.28.0 ([Giant Swarm app v1.3.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#v130-2020-01-30))
+
+- Updated from upstream `ingress-nginx` v0.26.1 - [changelog](https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.28.0).
+- Add metrics Service for prometheus-operator support.
```

Comment:
```suggestion
- Enabled monitoring by adding metrics Service for prometheus-operator support.
```

Would this make sense? It's tiny, not necessary.

Reply:
Not sure if it enabled monitoring, or it enhanced by maybe enabling to scape more metrics? Not sure the exact value...

Reply:
@giantswarm/team-celestial

Reply:
I don't know, but maybe @giantswarm/team-batman knows better?

Reply:
Halo owns I'd say optional/ondemand apps like nginx and Batman tooling/pipeline (for optional apps but also parts of kaas like delivery of built-in apps); Chiara is PO for whole apps area, so both Batman and Halo.

Anyway, added my suggestion https://github.com/giantswarm/releases/pull/84/files#r376336801 @cokiengchiara your suggestion got accepted/merged, please comment on my suggestion there if you agree with it or not. Resolving this one.